File: Topico_3/gangorra.hs
gangorra w x y z = 
  if w*x == y*z
    then 0
    else if w*x > y*z
      then -1
      else 1

main = do
    a <- readLn :: IO Int
    b <- readLn :: IO Int
    c <- readLn :: IO Int
    d <- readLn :: IO Int
    print $ gangorra a b c d

File: Topico_3/gangorra.js
gangorra = (p1, c1, p2, c2) => {
  if(p1*c1 == p2*c2){
    return 0
  }else if(p1*c1 > p2*c2){
    return -1
  }else{
    return 1
  }
}

console.log(gangorra(30,100,60,50));
console.log(gangorra(40,40,38,60));
console.log(gangorra(35,80,35,75));
console.log(gangorra(45,23,96,12));